src/postamble.js

```diff
@@ -303,7 +303,7 @@ function preInit() {
 // In MODULARIZE=instance mode we delay most of the initialization work until
 // the `init` function is called.
 export default async function init(moduleArg = {}) {
-  Module = moduleArg;
+  Object.assign(Module, moduleArg);
```

Comment:
I was really hoping we could move towards separating the arguments from the `Module` object.  I missed the change where we started assigning `var Module = {};`  in instance mode, but having it undefined helped point out places that were reading from the Module object before everything was setup.

Reply:
yeah I agree, but we have so much go that expects to be able to `Module[xxx]` in `--pre-js` code.  We would have to update all those tests if want to drop that.

I suppose we would make it conditional and only do this if there exists some pre-js code?